pennylane/gradients/vjp.py

```diff
@@ -137,9 +167,39 @@ def processing_fn(results):
 
 
 def batch_vjp(tapes, dys, gradient_fn, reduction="append"):
-    """Generate the gradient tapes and processing function required to compute
+    r"""Generate the gradient tapes and processing function required to compute
     the vector-Jacobian products of a batch of tapes.
 
+    Consider a function :math:`\mathbf{f}(\mathbf{x})`. The Jacobian is given by

```

Comment:
Instead of repeating the documentation here, maybe just add a line saying "see documentation of :func:`~.vjp`
for a theroetical description of the vector Jacobian product".

Reply:
I quite like the ability to give the info there and then 😆 

This is also text that doesn't have a high maintenance burdan, since it is explaining the math/concept, not the function.